service/src/main/python/jcarbon/tensorflow/callbacks.py

```diff
@@ -7,11 +7,14 @@
 from jcarbon.client import JCarbonClient
 from jcarbon.report import to_dataframe
 
+from jcarbon.nvml.sampler import create_report, NvmlSampler
+
 DEFAULT_PERIOD_MS = 10
 DEFAULT_PERIOD_SECS = 2
 DEFAULT_SIGNALS = [
     'nvml',
     'linux_process',
+    'linux_system',
```

Comment:
Don't add this here please. When you make a change to an existing part of an API, you are implicitly making everyone else's experiments have much bigger data (such as Harsha and Rushikesh). The intention is for you to pass a list of containing these elements to the signals argument in your callback instead of a sweeping change.

If you want to reuse those values, you can also do the following:

```
from jcarbon.tensorflow.callbacks import DEFAULT_SIGNALS

jcarb = JCarbonExperimentCallback(
    period_ms=args.sampling_period_millis,
    chunking_period_sec=args.collection_period_secs,
    signals=DEFAULT_SIGNALS + ['linux_system']
)
```

Reply:
oh that is really important, thanks for telling me. I will just add it in the callback